src/components/Profile.js

```diff
@@ -74,14 +76,23 @@ class Profile extends Component {
                                 value={this.state.name ? this.state.name : userinfo.name}
                                 onChange={this.onChange}
                                 label='Name'
-                                placeholder='Enter your name...' 
-                                inline
+                                placeholder='Enter your name...'
+                                fluid
+                            />
+                            <Form.Input
+                                name="zulip_id"
+                                value={userinfo.zulip_id}
+                                onChange={this.onChange}
+                                label='Zulip User ID'
+                                placeholder='Enter your Zulip user ID...' 
+                                fluid
+                                readOnly
```

Comment:
If it will be read-only then if a user enters the wrong zulip id at the first time then in the future he will not be able to change this.
What's your opinion on making it editable?

Reply:
🤔 could be editable. But I am not sure, what is all the context of this feature and the reasoning behind it. So we could later create an issue for it :) If this is good, lets merge it 🚀

Reply:
just need a 2nd approval here :)